src/components/profileAvatar/ProfileAvatar.ts

```diff
@@ -10,7 +13,42 @@ type TProfileAvatarState = {
 	popupClassName: string;
 };
 
-export class ProfileAvatar extends BaseComponent<TProfileAvatarState> {
+type TProfileAvatarBuildCtx = {
+	src: string;
+} | null;
+
+export class ProfileAvatar extends BaseComponent<
+	TProfileAvatarState,
+	null,
+	TProfileAvatarBuildCtx
+> {
+	private authController = new AuthController();
+	private isInitSrc = false;
+
+	private initSrc(): void {
+		const src = this.authController.getState()?.avatar;
+		if (src === null || src === "") {
+			this.build({
+				src: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=",
+			});
+		} else {
+			this.build({ src: getImgSrc(src) });
+		}
```

Comment:
```suggestion
        ...
        const defaultImage = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=";
        ...
	const src = this.authController.getState()?.avatar || defaultImage
	this.build({ src: getImgSrc(src) });
```
А что это за `getState` у контроллера? Для хранения состояния стоит завести стор и не мешать эту логику со слоем контроллеров